src-rs/lib.rs

```diff
@@ -1,20 +1,21 @@
 extern crate core;
 
-pub mod enums;
-mod models;
-mod services;
-mod utilities;
-
```

Comment:
This is multiple lines

Reply:
Good, easy-to-read top-level comment shows up as well, showing what I assume are only the lines you highlighted.